content/23.vaccines-novel.md

```diff
@@ -359,7 +374,7 @@ Self-replicating vaccines produce more viral antigens over a longer period of ti
 Finally, _in vitro_ dendritic cell non-replicating RNA vaccines limit transfection to dendritic cells.
 Dendritic cells are potent antigen-presenting immune cells that easily take up mRNA and present fragments of the translated peptide on their MHC proteins, which can then interact with T cell receptors.
 Ultimately, primed T follicular helper cells can stimulate germinal center B cells that also present the viral antigen to produce antibodies against the virus [@doi:10.1016/j.immuni.2014.10.004].
-These cells are isolated from the patient, grown and transfected _ex vivo_, and reintroduced to the patient [@doi:10.1038/nrd.2017.243].
+These cells are isolated from the patient, grown, and transfected _ex vivo_, and reintroduced to the patient [@doi:10.1038/nrd.2017.243].<!--HMR note: double-check with RL-->
```

Comment:
```suggestion
These cells are isolated from the patient, grown, and transfected _ex vivo_, and reintroduced to the patient [@doi:10.1038/nrd.2017.243].<!--HMR note: double-check with RL-->
```
@RLordan I am getting tripped up on this sentence -- I was initially reading it as they were both grown & transfected ex vivo, so it was a list of 3 items (isolated, grown/transfected, reintroduced.) Is that wrong?

Reply:
Thank you @RLordan! I changed the proposed edit above, hopefully this one is less of a weird sentence construction.